lambdas/process-async-cri-credential/src/test/java/uk/gov/di/ipv/core/processasynccricredential/pact/f2fCri/ContractTest.java

```
@@ -0,0 +1,768 @@
+                                asyncCriResponse.getVerifiableCredentialJWTs());
+
+                parsedJwts.forEach(
+                        credential -> {
+                            try {
+                                verifiableCredentialJwtValidator.validate(
+                                        credential, credentialIssuerConfig, TEST_USER);
+
+                                JsonNode credentialSubject =
+                                        objectMapper
+                                                .readTree(credential.getJWTClaimsSet().toString())
+                                                .get(VC)
+                                                .get(CREDENTIAL_SUBJECT);
+
+                                JsonNode nameParts =
+                                        credentialSubject.get(NAME).get(0).get(NAME_PARTS);
+                                JsonNode birthDateNode = credentialSubject.get(BIRTH_DATE).get(0);
+                                JsonNode passportNode = credentialSubject.get(PASSPORT).get(0);
+
+                                assertEquals("GivenName", nameParts.get(0).get(NAME_TYPE).asText());
+                                assertEquals(
+                                        "FamilyName", nameParts.get(1).get(NAME_TYPE).asText());
+
+                                assertEquals("Kenneth", nameParts.get(0).get(VALUE).asText());
+                                assertEquals("Decerqueira", nameParts.get(1).get(VALUE).asText());
+
+                                assertEquals("2030-01-01", passportNode.get(EXPIRY_DATE).asText());
+                                assertEquals(
+                                        "321654987", passportNode.get(DOCUMENT_NUMBER).asText());
+
+                                assertEquals("1965-07-08", birthDateNode.get(VALUE).asText());
+
+                            } catch (Exception ignored) {
+                            }
```

Can we ignore exceptions here and below? Wouldn't they indicate a test failure?